src/ogd/games/THERMOLAB/ThermoLabLoader.py

```diff
@@ -17,6 +16,8 @@
 from ogd.common.schemas.games.GameSchema import GameSchema
 from ogd.common.utils.utils import loadJSONFile
 from ogd.common.utils.Logger import Logger
+# import local files
+from . import features
```

Comment:
We'll also need a line for `from . import detectors`, I think.

Reply:
Nevermind, the detectors import is in there.
Also, this import is unused, we've got `from features import *`, so we don't need `from . import features`.